default.nix

```diff
@@ -63,6 +63,19 @@ let
         buildInputs = [ self_.gfortran ];
       });
 
+      fftw-mpi = self.fftw.overrideDerivation (oldAttrs: {
```

Comment:
I think that should be `overrideAttributes` here (also for the override of `fftw`. I probably made a mistake here using `overrideDerivation` in the first place).

Reply:
Actually FFTW does not export overrideAttributes. I was also a little bit surprised but I have tried after your first comment.

Reply:
My bad, I meant `overrideAttrs`. I fixed it on master. Just do a rebase.

Reply:
Done